### DISCUSSION TITLE
clip-path: path()

### DISCUSSION THREAD

**Discussion by Nis3r** [cssbattle/forum#278] [Q&A]
When I go to use clip-path I get an alert "SVGs are not allowed"
Are we allowed to use clip-path?
The top scores on the few open book solutions use clip-path so that's why I was trying to learn it.

**Comment by gunnarbittersmann** [comment]
No, we are not allowed to use `clip-path` with `path()`. 🤣
But you can’t forbid what you can’t control. There’s a hack which has been made public through open targets.
IMHO, the restriction should be removed so we don’t need to escape.

**Reply by Nis3r** [reply]
ok thank you! 😃
Without being able to see most solutions it's hard to know if I'm solving them in the way that was intended! 😛
I think I'm using margin more than was intended so I want to try finding more creative solutions 🙂
